Gruntfile.js

```diff
@@ -59,6 +59,44 @@ module.exports = function(grunt) {
           {expand: true, cwd: 'test/fixtures/', src: ['**'], dest: './'}
         ]
       },
+      zipCreateEmptyArchiveTrue: {
+        options: {
+          createEmptyArchive: true,
+          archive: function () {
+            return 'tmp/zip_must_be_created_1.zip';
+          }
+        },
+        files: [{
+          expand: true,
+          cwd: 'test/fixtures/',
+          src: ['NotExistingFilePath.js']
+        }]
+      },
+      zipCreateEmptyArchiveFalse: {
+        options: {
+          createEmptyArchive: false,
+          archive: function () {
+            return 'tmp/zip_should_not_be_created.zip';
+          }
+        },
+        files: [{
+          expand: true,
+          cwd: 'test/fixtures/',
+          src: ['NotExistingFilePath.js']
+        }]
+      },
+      zipNoCreateEmptyArchiveOption: {
+        options: {
+          archive: function () {
+            return 'tmp/zip_must_be_created_1.zip';
```

Comment:
I don't see any other file named `zip_must_be_created_1.zip` so maybe change all instances to `zip_must_be_created.zip`?